### PR TITLE
[SMSDEV-2478] Rollback test-connection and save configuration endpoints' 400 error response 

### PR DESCRIPTION
**Pull request by matusekma**
Co-authored-by: epgrubmair <peter.grubmair@emarsys.com>

### CODE REVIEW

File: openapi/sms/v1/partner-service.yaml [hunk 1]
```diff
@@ -376,13 +354,28 @@ components:
...
         - $ref: '#/components/schemas/OutboundMessageBatchMessageSpecificError'
+  responses:
+    ClientConfigurationInvalidFormatResponse:
+      description: Request body is invalid and does not follow API specification.
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/InvalidFieldErrors'
+    TestConnectionInvalidFormatResponse:
```
Comment (emarsys-ziyad) [inline openapi/sms/v1/partner-service.yaml]: we only need 1 response schema here, they are both identical, how about calling it `InvalidClientConfigurationFormatResponse`?

**Review by emarsys-ziyad — CHANGES_REQUESTED** [review]

File: openapi/sms/v1/partner-service.yaml (hunk 1 above)
Comment (matusekma) [reply openapi/sms/v1/partner-service.yaml]: We were thinking about doing it like this for the sake of extensibility and code generation. But if you insist on making it uniform, I am okay with that.

Comment (emarsys-ziyad) [reply openapi/sms/v1/partner-service.yaml]: i imagine the testing of the connection setup and saving of the connection setup would always have a similar payload therefore similar validation error, or maybe you imagine something else?  it's kind of like a preview of if the save would be successful

Comment (matusekma) [reply openapi/sms/v1/partner-service.yaml]: Fair point, we will merge them.

**Review by emarsys-ziyad — APPROVED** [review]

**Pull request merged**